codepipeline_deploy.sh

```diff
@@ -12,6 +11,22 @@ if [ -z "${PIPELINE_TIMEOUT}" ]; then
     PIPELINE_TIMEOUT=1800
 fi
 
+# Get the HEAD commit ID for version.json in master branch if exists
+MASTER_COMMIT_ID=`aws codecommit get-file --repository-name AppManifest-"$SA_NAME"-"$ROS_DISTRO"-gazebo"$GAZEBO_VERSION" --file-path version.json`
```

Comment:
Can you avoid such long lines here and in the commands below? You can use `\` (with no space) at the end of the line for that

Reply:
I have uploaded a new version that doesn't use codecommit anymore. Please take a look.

Reply:
Here is a sucessfully build and end-to-end test in beta env: https://travis-ci.org/aws-robotics/aws-robomaker-sample-application-helloworld/jobs/517940836